tests/ci/cdk/cdk/codebuild/github_ci_android_omnibus.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+# Doc for batch https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list
```

Comment:
* Try to reuse existing omnibus.yml to avoid a new webhook creation?
* How long does each dimension take?
* How's the memory/CPU usage of `BUILD_GENERAL1_LARGE`? This info is available through metrics tab of the build job. If the usage is low, smaller instance type can be used but remember some timeout cases.

Reply:
update: the android apk packaging at the end of the build downloads and runs some jar files which `BUILD_GENERAL1_SMALL` seems to not be able to handle well, and fails. I'll be using `BUILD_GENERAL1_MEDIUM` instead.

Reply:
> Try to reuse existing omnibus.yml to avoid a new webhook creation?

I revisited this thought and think it makes sense to have a separated stack/webhook which will have device farm relevant resource definition.